legend-engine-xt-sql-query/src/main/java/org/finos/legend/engine/query/sql/model/SchemaColumn.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package org.finos.legend.engine.query.sql.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class SchemaColumn
+{
+    private final String name;
+    private final String type;
```

Comment:
type as a string...Is there an enum domain we can use?

Reply:
The type is derived from TDS Column, which has the pure `DataType` which can be primitive (only primitive for now), or other pure classes (in the future). The pure `DataType` is converted to string using the pure function `meta::pure::functions::meta::elementToPath`

There is a datatype enum defined in persistence [model](https://github.com/finos/legend-engine/blob/master/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/logicalplan/datasets/DataType.java), and an abstract class in [relational store](https://github.com/finos/legend-engine/blob/master/legend-engine-xt-relationalStore-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/store/relational/model/datatype/DataType.java). But there is no straightforward way to convert a pure `DataType` to either of those